src/desktop/apps/identity_verification/client.tsx

```diff
@@ -0,0 +1,30 @@
+import { buildClientApp } from "reaction/Artsy/Router/client"
+import { data as sd } from "sharify"
+import { loadableReady } from "@loadable/component"
+
+import { routes } from "reaction/Apps/IdentityVerification/routes"
+import React from "react"
+import ReactDOM from "react-dom"
+
+const mediator = require("desktop/lib/mediator.coffee")
+
+buildClientApp({
+  routes,
+  context: {
+    user: sd.CURRENT_USER,
+    mediator,
+    isEigen: sd.EIGEN,
```

Comment:
```suggestion
    isEigen: sd.EIGEN,
```

we'll likely want to remove as it's included in all calls to `buildClientApp`